### ISSUE TITLE
Redirection of screenshots

### ISSUE THREAD

**Issue by mimmiepower** [gyng/save-in#145]
It would be nice if this extension could redirect screenshots saved with Firefox's "Take a Screenshot" feature.

**Comment by gyng** [comment]
Yeah, this is something I would love too. I took a quick look but didn't think of any easy way of doing it.
